### PR TITLE
Fixed UI tests

### PR DESCRIPTION
**Pull request by sghai**
- blocked test_key_associate_11 with bz-id 1210180
- updated param names for two tests
- Fixed `test_delete_user`: Here we are searching the entity after deletion and success notification too. But test sometime fails when selenium unable to capture the success notification.

### CODE REVIEW

**Comment by sghai** [comment]
Fixed locator to update name/description of Life-cycle env (`test_positive_update_content_environment`)

**Comment by sthirugn** [comment]
ACK

File: robottelo/ui/repository.py [hunk 1]
```diff
@@ -40,6 +40,9 @@ def create(self, name, product=None, gpg_key=None, http=False, url=None,
                 Select(type_ele).select_by_visible_text(gpg_key)
             if url:
                 self.text_field_update(locators["repo.url"], url)
+            if upstream_repo_name:
```
Comment (Ichimonji10) [inline robottelo/ui/repository.py]: It's probably a bit nicer to write `if upstream_repo_name is not None:`. However, I see that this idiom of `if method_argument:` is used quite a bit, so I'm not going to block on account of this.

**Comment by Ichimonji10** [comment]
ACK

**Pull request merged**